images/openshift-enterprise-node.yml

```diff
@@ -28,5 +22,6 @@ labels:
   io.k8s.display-name: OpenShift Container Platform Node
   io.openshift.tags: openshift,node
   vendor: Red Hat
+# TODO: this will become openshift/ose-sdn in 4.1
```

Comment:
Please note: Comments are frequently removed by automatic reprocessing of the data files. I won't reject this due to the comment, but we make no effort to keep comments if the data is reprocessed.

Reply:
That’s fine, I couldn’t tell whether you preserved them.